modules/instrument_builder/jsx/react.instrument_builder.js

```diff
@@ -76,7 +78,8 @@ var LoadPane = React.createClass({
               <span aria-hidden="true">&times;</span>
             </button>
             <strong>Error!</strong><br/>
-            Instrument file can not contain elements with identical name! <br/>
+            {this.state.alertMessage} <br/>
+            Instrument file can not contain elements with identical names! <br/>
```

Comment:
Since you added the ability to pass messages through callback, I think the whole message should be passed so it is all in one place.